### PR TITLE
ZGto2LG_amcatnloFXFX-pythia8

### PR DESCRIPTION
**Pull request by sihyunjeon**
@agrohsje 

Added filters at LHE level for pTG

### CODE REVIEW

**Comment by sihyunjeon** [comment]
should we add "'BeamRemnants:primordialKThard = 2.48'" for ZG as well?

File: Cards/MadGraph5_aMCatNLO/Others/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "3",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massZ.dat",
+    "model_params_user": [
+        "set run_card ptj 10",
+        "set run_card mll_sf 50.0",
+        "set run_card ptgmin 70.0"
```
Comment (agrohsje) [inline Cards/MadGraph5_aMCatNLO/Others/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8.json:13]: This is 70 while the name says 100to200.

File: Cards/MadGraph5_aMCatNLO/Others/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+    "template": "NLO_run_card.dat",
+    "template_user": [],
+    "template_vars":{
+        "parton_shower": "pythia8",
+        "ickkw": "3",
+        "maxjetflavor": "5"
+    },
+    "model_params": "massZ.dat",
```
Comment (agrohsje) [inline Cards/MadGraph5_aMCatNLO/Others/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8/ZGto2LG-1Jets_MLL-50_PTG-100to200_amcatnloFXFX-pythia8.json:9]: Just to confirm, for every sample involving a Z (including ZW, we are using Z.dat)?

File: Fragments/Filter/LHE_Gamma_Pt100to200_filter.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
+LHE_Gamma_Pt100to200_filter = cms.EDFilter("LHE_Gamma_Pt100to200_filter",
```
Comment (agrohsje) [inline Fragments/Filter/LHE_Gamma_Pt100to200_filter.dat:1]: The specifiction in EDFilter("blabla") blabla should correspond to the official name of the filter, not your name, no ?

Comment (sihyunjeon) [reply Fragments/Filter/LHE_Gamma_Pt100to200_filter.dat]: fixed

**Pull request merged**